### DISCUSSION TITLE
Connecting with 1.19.3/4 to a 1.19.2 server

### DISCUSSION THREAD

**Discussion by AstroSquared** [Minestom/Minestom#1805] [Help] [answered]
Hello,

I'm trying to use Velocity + ViaVersion to allow connections from 1.19.2 to 1.19.4 on a 1.19.2 Minestom server.
Right now, when I join on the Minestom server version (.2), it works fine, but connecting with a newer version (.3) causes an error that says "Invalid Version, please use 1.19.2". I found this in the Minestom code, so I know the proxy is letting the player join.

Am I doing something wrong, or am I not understanding how ViaVersion works?

tl;dr connecting to minestom with a velocity+viaversion proxy with a newer version doesn't work

**Comment by p3sto** [comment]
Try using this [tool](https://jo0001.github.io/ViaSetup/?lang=en). It should help with the setup.

**Reply by AstroSquared** [reply]
This doesn't change anything, it just tells me to block versions less then 1.19.1. Issue still persists.

**Comment by AstroSquared** [comment, marked as answer]
Turns out I had a skill issue moment and made the 1.19.3 client connect directly to the minestom server without going through the proxy. Always double check your server AND your client!
